plugins/zenoh-plugin-trait/src/manager.rs

```diff
@@ -60,6 +62,11 @@ impl<StartArgs: PluginStartArgs, Instance: PluginInstance> PluginStatus
     fn name(&self) -> &str {
```

Comment:
I'm not sure that after this update we need this `name` getter at all. If plugin is dynamic, the `path()` getter gives us full information where this plugin came from. If it's static, name and id is the same thing.

Reply:
The former is true, while not the latter, the user can statically link the plugin once, but instantiate it multiple times with different IDs.